src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -83,8 +87,6 @@ object Config {
 
     val chainId: Byte = Hex.decode(blockchainConfig.getString("chain-id")).head
 
-    val HomesteadBlock: BigInt = 1150000
-
     // YP eq 150
     val BlockReward = UInt256(BigInt("5000000000000000000"))
```

Comment:
@LukasGasior1 even if this is defined in the YP, do you think it's worthy to put it inside a Config (not EVMConfig)?